src/JsFunctionsScanner.php

```diff
@@ -152,4 +151,44 @@ public function saveGettextFunctions( Translations $translations, array $options
 
 		$traverser->traverse( $ast );
 	}
+
+	/**
+	 * Resolve the callee of a call expression using known formats.
+	 *
+	 * @param Node\CallExpression $node The call expression whose callee to resolve.
+	 *
+	 * @return Node\Identifier|bool The identifier if resolved. False if not.
+	 */
+	private function resolveExpressionCallee( Node\CallExpression $node ) {
+		// If the callee is a simple identifier it can simply be returned.
+		// For example: __( "translation" ).
+		if ( 'Identifier' === $node->getCallee()->getType() ) {
+			return $node->getCallee();
+		}
+
+		// If the callee is a member expression resolve it to the property.
+		// For example: wp.i18n.__( "translation" ) or u.__( "translation" ).
+		if (
+			'MemberExpression' === $node->getCallee()->getType() &&
+			'Identifier' === $node->getCallee()->getProperty()->getType()
+		) {
+			return $node->getCallee()->getProperty();
+		}
+
+		// If the callee is a call expression as created by WebPack resolve it.
+		// For example: Object(u.__)( "translation" ).
+		if (
+			'CallExpression' ===  $node->getCallee()->getType() &&
+			'Identifier' ===  $node->getCallee()->getCallee()->getType() &&
+			'Object' === $node->getCallee()->getCallee()->getName() &&
+			! empty( $node->getCallee()->getArguments() ) &&
```

Comment:
This fails on PHP 5.4: https://travis-ci.org/wp-cli/i18n-command/jobs/446078984#L633

`empty()` needs to work on a reference for PHP 5.4 and lower, so you cannot directly use a method call here.

Reply:
Alternatively, you can change to test to not use `empty()`, but compare to an empty array (which I think is the expected return value here).